raft/src/test/java/org/apache/kafka/raft/internals/BatchMemoryPoolTest.java

```diff
@@ -104,4 +128,15 @@ public void testReleaseBufferNotMatchingBatchSize() {
         assertThrows(IllegalArgumentException.class, () -> pool.release(buffer));
     }
 
+    private ByteBuffer touch(ByteBuffer buffer) {
```

Comment:
nit: `touch` seems a little vague. I think we're just trying to simulate some buffer usage?

Reply:
Renamed the function to `update`.